<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Kiran-PC" failures="1" tests="1" name="Test1" time="346.766" errors="0" timestamp="2023-04-14T18:56:50 IST">
  <testcase classname="organization.CreateOrganizationTest" name="createOrganization" time="50.546">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at organization.CreateOrganizationTest.createOrganization(CreateOrganizationTest.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- createOrganization -->
</testsuite> <!-- Test1 -->
